ibis/spark/tests/conftest.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+import ibis
+
+
+@pytest.fixture(scope='session')
+def client():
+    pytest.importorskip('pyspark')
+    client = ibis.spark.connect()
+    df = client._session.createDataFrame([(1, 'a')], ['foo', 'bar'])
+    df.createOrReplaceTempView('simple')
+    df1 = client._session.createDataFrame(
+        [
+            (
+                [1, 2],
+                [[3, 4], [5, 6]],
+                {(1, 3) : [[2, 4], [3, 5]]},
+            )
+        ],
+        ['c1', 'c2', 'c3']
```

Comment:
Can you make more descriptive column names?

Reply:
Like `list_of_ints`, `list_of_list_of_ints`, and `map_tuple_list_of_list_of_ints`. I know it looks cumbersome but it will be easier to read in the tests that use the `nested_types` fixture.